api-specs/Gateway-EE/3.4/kong-ee-3.4.yaml

```diff
@@ -8505,37 +8505,39 @@ servers:
     name: Consumers
-  - description: Certificates
+  - description: A certificate object represents a public certificate, and can be optionally paired with the corresponding private key. These objects are used by Kong to handle SSL/TLS termination for encrypted requests, or for use as a trusted CA store when validating peer certificate of client/service. Certificates are optionally associated with SNI objects to tie a cert/key pair to one or more hostnames.If intermediate certificates are required in addition to the main certificate, they should be concatenated together into one string.
     name: Certificates
-  - description: SNIs
+  - description: An SNI object represents a many-to-one mapping of hostnames to a certificate. That is, a certificate object can have many hostnames associated with it; when Kong receives an SSL request, it uses the SNI field in the Client Hello to lookup the certificate object based on the SNI associated with the certificate.
     name: SNIs
-  - description: Targets
+  - description: A target is an ip address/hostname with a port that identifies an instance of a backend service. Every upstream can have many targets, and the targets can be dynamically added, modified, or deleted. Changes take effect on the fly.
     name: Targets
-  - description: CA certificates
+  - description: A CA certificate object represents a trusted CA. These objects are used by Kong to verify the validity of a client or server certificate.
     name: CA Certificates
-  - description: Upstreams
+  - description: The upstream object represents a virtual hostname and can be used to loadbalance incoming requests over multiple services (targets).An upstream also includes a [health checker](https://docs.konghq.com/gateway/3.4.x/how-kong-works/health-checks/), which is able to enable and disable targets based on their ability or inability to serve requests. The configuration for the health checker is stored in the upstream object, and applies to all of its targets.
     name: Upstreams
-  - description: Vaults
+  - description: Vault objects are used to configure different vault connectors. Examples of vaults are environment variables, Hashicorp Vault and AWS Secrets Manager. Configuring a vault allows referencing the secrets with other entities. For example a certificate entity can store a reference to a certificate and key, stored in a vault, instead of storing the certificate and key within the entity. This allows a proper separation of secrets and configuration and prevents secret sprawl. Secrets rotation can be managed using [TTLs](https://docs.konghq.com/gateway/latest/kong-enterprise/secrets-management/advanced-usage/).
     name: Vaults
-  - description: Keys
+  - description: A Key object holds a representation of asymmetric keys in various formats. When Kong or a Kong plugin requires a specific public or private key to perform certain operations, it can use this entity.
```

Comment:
```suggestion
  - description: |
      A key object holds a representation of asymmetric keys in various formats. When Kong Gateway or a Kong plugin requires a specific public or private key to perform certain operations, it can use this entity.
```